File: Programmers/practice1.py
def solution(s):
    answer = True
    p_count = 0
    y_count = 0

    for c in s:
        if c == 'p' or c == 'P':
            p_count += 1
        if c == 'y' or c == 'Y':
            y_count += 1

    if p_count == y_count:
        answer = True
    else:
        answer = False

    return answer

s = "pPoooyY"
# answer = True
print(solution(s))

s = "Pyy"
# answer = False
print(solution(s))

File: Programmers/practice1.js
function solution(arr) {
  var answer = arr.reduce((total, item) => total + item, 0) / arr.length;

  return answer;
}

let arr = [1, 2, 3, 4];
// answer = 2.5
console.log(solution(arr));

arr = [5, 5];
// answer = 5
console.log(solution(arr));
